src/compiler/sys.ts

```diff
@@ -394,9 +394,7 @@ namespace ts {
                         }
                     );
                 },
-                resolvePath: function(path: string): string {
-                    return _path.resolve(path);
-                },
+                resolvePath: path => _path.resolve(path),

```

Comment:
:-1: - that's a possible behavioral change in multiple ways (different `this` reference, different behavior when more than one argument is supplied)